usagov/usagov-2021
# USAGOV-501-Persistant-Carousel: Setting the carousel index with sessi…

…on storage


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-501

## Description

Created a persistent index variable for the carousel so that when users leave the homepage and come back it's on the same card. Used session variable instead of a local variable because their time to live is shorter. If a user comes back to the homepage in a different window or tab the carousel will be back to the 0 index.

See [homepage PR ](https://github.com/usagov/usagov-2021/pull/631)for set up.

### Testing Instructions
After setup (see above) scroll down to carousel. 
Focus on a card using either the next button or circle buttons. 
Click to a link out of the homepage. 
Come back to homepage. 
Selected card should be visible and correct circle filled. 

## Checklist for the Developer



## Checklist for the Peer Reviewers
